owa/src/features/stockItems/StockItemBatchInformation.tsx

```diff
@@ -61,7 +61,7 @@ const StockItemBatchInformationTable: React.FC<StockItemBatchInformationTablePro
       location: row?.partyName,
       quantity: row?.quantity?.toLocaleString() ?? "",
       batchno: row.batchNumber ?? "",
-      quantityuom: row.quantityUoM ?? ""
+      quantityuom: `${row.quantityUoM ?? ""} of  ${row.quantityFactor ?? ""}`
```

Comment:
``${row.quantityUoM ?? ""} ${row?.quantityFactor ? of ${row?.quantityFactor} : ""}``